src/explorer/SiteTreeItem.ts

```diff
@@ -182,4 +192,15 @@ export abstract class SiteTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
             telemetryProperties.promptToSaveDeployConfigs = 'Skip for now';
         }
     }
+
+    private getIgnoredFoldersForDeployment(runtime: string): string[] {
+        switch (runtime) {
+            case runtimes.node:
+                return ['node_modules{,/**}'];
+            case runtimes.python:
+                return ['env/{,/**}', 'venv{,/**}', 'lib{,/**}', 'lib64{,/**}'];
+            default:
```

Comment:
Please file an issue and/or send an email to the python team about the non-virtual environment values as well. You picked two values (lib and lib64) out of a pretty long list from the "# Distribution / packaging" section and it still seems arbitrary. I also think we might want to include the values from the "# Byte-compiled / optimized / DLL files" section